src/nav/nav_model/nav_model_rings_simulated.py

```
@@ -0,0 +1,229 @@
+
+        with self._logger.open(f'CREATE SIMULATED RINGS MODEL FOR: {self._ring_name}'):
+            self._create_model(always_create_model=always_create_model,
+                               never_create_model=never_create_model,
+                               create_annotations=create_annotations)
+
+        end_time = now_dt()
+        metadata['end_time'] = end_time.isoformat()
+        metadata['elapsed_time_sec'] = (end_time - start_time).total_seconds()
+
+    def _create_model(self,
+                      always_create_model: bool,
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)